examples/flutter_gallery/test/smoke_test.dart

```diff
@@ -43,8 +43,7 @@ Future<Null> smokeDemo(WidgetTester tester, String routeName) async {
   Finder backButton = find.byTooltip('Back');
   expect(backButton, findsOneWidget);
   await tester.tap(backButton);
-  await tester.pump(); // Start the pop "back" operation.
-  await tester.pump(const Duration(seconds: 1)); // Wait until it has finished.
+  await tester.pumpUntilNoTransientCallbacks(const Duration(seconds: 1));
```

Comment:
we should be careful about using pumpUntilNoTransientCallbacks. It can hide bugs. In particular here I think it would be better to just have the two pumps, because then we're clear about how many frames we're expecting. Why would we be getting more?